tests/NCronJob.Tests/TestHelper.cs

```diff
@@ -201,9 +201,7 @@ public sealed class Storage
 #else
     private readonly object locker = new();
 #endif
-#pragma warning disable CA1002 // Do not expose generic lists
-    public List<string> Entries { get; private set; } = [];
-#pragma warning restore CA1002 // Do not expose generic lists
+    public IList<string> Entries { get; private set; } = [];
```

Comment:
Dammit, you're right.

I forgot the index was exposed by the interface and not the concrete type. Good catch!

Reply:
Nah no worries.